### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by hugohabel**
Basic logic to retrieve, parse and store airports information.

### CODE REVIEW

File: utils/parsing.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ */
+function extractIdent(codes) {
+  if (!codes || codes.length === 0) {
+    logger.log('Error → Value undefined or no information provided', 'error');
+  }
+  var icaoCodes = [];
+
+  for (var i = 0, l = codes.length; i < l; i++) {
+    logger.log('ICAO code → ' + codes[i].ident + ' - ' + codes[i].name, 'info');
+    icaoCodes.push(codes[i].ident);
```
Comment (frankyntin) [inline utils/parsing.js]: THis looks to me a job for a .map function

File: utils/parsing.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+/**
+ * Extract the relevant information from the list of frequencies
+ *
+ * @param frequencies Array List of frequencies
+ * @returns {Array} List of frequencies with the type and the frequency in mhz
+ */
+function parseFrequencies(frequencies) {
+  var airportFrequencies = [];
+
+  for (var i = 0, l = frequencies.length; i < l; i++) {
```
Comment (frankyntin) [inline utils/parsing.js]: .map function

File: utils/parsing.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    logger.log('Frequency → ' + frequencies[i].airport_ident + ' - ' + frequencies[i].type, 'info');
```
Comment (frankyntin) [inline utils/parsing.js]: Seems like literal template string might be useful here

File: utils/parsing.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+/**
+ * Extract the relevant information from the list of runways
+ *
+ * @param runways Array List of runways
+ * @returns {Array} List of runways with relevant information
+ */
+function parseRunways(runways) {
+  var airportRunways = [];
+
+  for (var i = 0, l = runways.length; i < l; i++) {
```
Comment (frankyntin) [inline utils/parsing.js]: .map

File: utils/parsing.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    logger.log('Runway → ' + runways[i].airport_ident, 'info');
+
+    airportRunways.push({
```
Comment (frankyntin) [inline utils/parsing.js]: Why don't you use a constructor?

File: utils/parsing.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+      he_elevation_ft: runways[i].he_elevation_ft
+    });
+  }
+
+  return airportRunways;
+}
+
+// Exports
+
+module.exports = {
```
Comment (frankyntin) [inline utils/parsing.js]: I'd rather used named exports

File: utils/indexManagement.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+/**
+ * Store updated index to file
+ *
+ * @param index   Int   Current index
+ * @returns {Promise}
+ */
+function setCurrentIndex (index) {
+  return new Promise(function (resolve, reject) {
```
Comment (frankyntin) [inline utils/indexManagement.js]: Not a fan of annon functs, in which case I'd suggest to rather use arrow funcs

File: dataLoader/dataLoader.js [hunk 8]
```diff
@@ -0,0 +1,252 @@
+/**
```
Comment (frankyntin) [inline dataLoader/dataLoader.js]: In general I think this file contains two different type of handlers. 
1 type is related to DB connection either for retrieving or writing info
2 type related to manipulate data retrieved or stored on DB.

As for separation the concerns is easy to understand the purpose of a single class/file if these are clearly split.

File: dataLoader/dataLoader.js [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+// MySQL query to get ICAO codes
+var icaoCodesQuery = 'SELECT ident, name FROM airport ORDER BY id LIMIT ';
+// MySQL query to get frequencies for given ICAO code
+var getFrequenciesQuery = 'SELECT * FROM frequency WHERE airport_ident = ';
+// MySQL query to get runways for given ICAO code
+var getRunwaysQuery = 'SELECT * FROM runway WHERE airport_ident = ';
+
+// MySQL create connection
+var connection = mysql.createConnection(config.mySql);
+connection.connect();
```
Comment (frankyntin) [inline dataLoader/dataLoader.js]: Following a singleton pattern this can be reused by other instances without need to open a connection each time

Comment (hugohabel) [reply dataLoader/dataLoader.js]: MySQL connection is opened just one time per script execution, then closed at the end when all information has been retrieved, just before sending it to MongoDB.

**Pull request merged**